packages/docs/integrations/destinations/segment.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Segment Destination"
```

Comment:
Hey if you're going to make all of these changes to the titles in the docs, can you do it in a separate PR? The only reason I'm hesitant to make these changes is that I believe they will negatively impact the search-ability of the docs. For example, if you search for the term "segment" in the docs, because it's an overloaded term, you'll see a number of unrelated pages, on segment nodes, segment.io etc.  By including the word "Destination" in the title, we allow users to distinguish between this segment page and others in the search results.